.renovaterc.json

```diff
@@ -21,7 +21,7 @@
 
   "lockFileMaintenance": {
     "enabled": true,
-    "schedule": "before 5am on monday"
+    "schedule": "before 5am"
```

Comment:
With this schedule, we will have a new pull request almost every day. Are we fine with having our git commit history polluted with 20-30 "update package-lock" commits every months?

Reply:
let me revert the schedule change and create a new PR to find a more reasonable schedule.